### PR TITLE
Replace multi-line ScopedBlocks with Tabs

### PR DESCRIPTION
**Pull request by ptgott**
Contributes to #30268

The documentation scope switcher tends to confuse users, and the ScopedBlock component hides docs content based on the scope switcher. To help remove ScopedBlocks from the docs site, this change replaces multi-line ScopedBlocks with Tabs components if they include variations for different scopes.

While most multi-line ScopedBlocks function like Tabs, there are a few edge cases, which this change addresses individually.

Note that this change does not intend to remove all ScopedBlocks that are placed inline within a paragraph. That will be the goal of a separate change.

### CODE REVIEW

**Comment by ptgott** [comment]
**TODO:** I am going to backport this manually

**Review by zmb3 — APPROVED** [review]

**Review by lsgunn-teleport — APPROVED** [review]
for the sake of visibility in the near term.

File: docs/pages/includes/tctl.mdx [hunk 1]
```diff
@@ -17,9 +18,12 @@ $ tctl status
 You can run subsequent `tctl` commands in this guide on your local machine.
 For full privileges, you can also run `tctl` commands on your Teleport Auth Service host.
 
-</ScopedBlock>
+</TabItem>
+</Tabs>
```
Comment (greedy52) [inline docs/pages/includes/tctl.mdx:22]: I don't think this end `</Tabs>` intended. Right?

Comment (greedy52) [reply docs/pages/includes/tctl.mdx:22]: Screenshot 2023-08-18 at 2 04 50 PM

Comment (ptgott) [reply docs/pages/includes/tctl.mdx:22]: @greedy52 Thanks for catching this! I used a script to replace `ScopedBlock`s with `Tabs` components. I thought I caught all of the edge cases, but there are a couple remaining that I'll fix.

**Pull request merged**